Snippets/Core/Core_6/UpgradeGuides/5to6/DisableRetries.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Core6.UpgradeGuides._5to6
+{
+    using NServiceBus;
+
+    public class DisableRetries
+    {
+        DisableRetries(EndpointConfiguration endpointConfiguration)
+        {
+            #region 5to6-DisableRetries
+
+            endpointConfiguration.DisableFirstLevelRetries();
+            endpointConfiguration.SecondLevelRetries().Disable();
```

Comment:
It just seems odd that for disabling SLR and FLR, the API is completely different.